python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -1664,3 +1664,124 @@ def forward(self, x):
 @register_test_case(module_factory=lambda: ToCopyWithDTypeFalsePinMemoryModule())
 def ToCopyWithDTypeFalsePinMemoryModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 2, 4))
+
+# ==============================================================================
+
+class MaxPool2dWithIndicesBackwardModule(torch.nn.Module):
```

Comment:
nit: `MaxPool2dWithIndicesBackwardStatic4DModule` to match `MaxPool2dWithIndicesBackwardDynamic4DModule` below.